src/woo_publications/publications/tests/test_publications_api.py

```diff
@@ -1342,12 +1342,12 @@ def test_create_publication(self):
                 "verkorteTitel": "one",
                 "omschrijving": "Lorem ipsum dolor sit amet, "
                 "consectetur adipiscing elit.",
-                # These values will be overwritten
-                "bronBewaartermijn": "THIS VALUE WON'T BE USED",
-                "selectiecategorie": "THIS VALUE WON'T BE USED",
+                # These values will not be overwritten because the status is concept
+                "bronBewaartermijn": "THIS VALUE WILL BE USED",
+                "selectiecategorie": "THIS VALUE WILL BE USED",
```

Comment:
I don't have the domain knowledge to asses this.

I expected this to already be used, because I thought you either don't specify these fields and the will be applied from the informatieCategorieen defaults, or you do specify them, and they get used.

Reply:
not when you create a published publication, right? Because then the calculation method kicks in and overwrites the api-client supplied values.